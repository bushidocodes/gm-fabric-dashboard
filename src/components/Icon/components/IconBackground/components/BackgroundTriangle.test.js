import React from "react";
import { shallow } from "enzyme";
import BackgroundTriangle from "./BackgroundTriangle";

describe("BackgroundTriangle", () => {
  it("matches snapshot", () => {
    const aBackgroundTriangle = shallow(<BackgroundTriangle />);
    expect(aBackgroundTriangle).toMatchSnapshot();
  });
});
