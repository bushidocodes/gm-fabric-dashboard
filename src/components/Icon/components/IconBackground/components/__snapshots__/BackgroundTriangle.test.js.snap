// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundTriangle matches snapshot 1`] = `
<g
  className="iconBackground"
  fill="transparent"
  fillOpacity="1"
  fillRule="evenodd"
  id="Square"
>
  <polygon
    id="Triangle-6"
    points="12 1 24 20 0 20"
  />
</g>
`;
